src/core/server/saved_objects/migrations/actions/wait_for_index_status_yellow.ts

```diff
@@ -37,7 +42,10 @@ export const waitForIndexStatusYellow =
     client,
     index,
     timeout = DEFAULT_TIMEOUT,
-  }: WaitForIndexStatusYellowParams): TaskEither.TaskEither<RetryableEsClientError, {}> =>
+  }: WaitForIndexStatusYellowParams): TaskEither.TaskEither<
+    RetryableEsClientError | IndexNotYellowTimeout,
```

Comment:
Can this still return a `RetryableEsClientError`?

Reply:
yeah we still have `.catch(catchRetryableEsClientErrors)` which will return a `RetryableEsClientError` for e.g. a 503 status code.